_data/authors.yml

```diff
@@ -214,9 +214,9 @@
   spiel: |
       <a href="/blog/authors/keane/">Keane</a> is a Data Scientist at Snowplow. You can find him on <a href="https://uk.linkedin.com/in/keanerobinson">LinkedIn</a>
 
-- name: Kosta
+- name: Kostas
```

Comment:
Not sure how this works, but doesn't this have to match the author here: pages/blog/authors/kostas.html ?